tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -85,7 +85,7 @@ then
   echo "Setting default ruby version."
   rvm use 2.7.0 --default
   echo "Installing cocoapods."
-  time gem install cocoapods --version 1.3.1 --no-document
+  time gem install cocoapods --version 1.3.1 --no-document --user-install
```

Comment:
as a followup we should check whether this step is required for ruby deps at all.